src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
```

Comment:
So to be correct it would need to replace `declare(strict_types=1,tick=1);` by `declare(tick=1);`, right?